src/components/Campaign.tsx

```diff
@@ -30,9 +34,8 @@ export const Campaign = () => {
 
   const onSave = async () => {
     const design = await getDesign();
-    const html = await getHtml();
-    console.log("Saving design", design);
-    console.log("Saving html", html);
+    const htmlContent = await getHtml();
+    campaignContentMutation.mutate({ idCampaign, design, htmlContent });
```

Comment:
Por acá se soluciona [DE-449](https://makingsense.atlassian.net/browse/DE-449) simplemente pasando en la _mutation_ el meta
```suggestion
    campaignContentMutation.mutate({ idCampaign, meta: design, htmlContent });
```

Reply:
Yo haría ese cambio en el `HtmlEditorApiClientImpl`, en mi opinión, para casi toda la webapp se llama `design` (no `meta`) y me parece que lo mejor es llevar ese mapeo lo más abajo posible.

Reply:
Entiendo lo abordaré desde esa perspectiva.